engines/sci/graphics/frameout.cpp

```diff
@@ -1334,14 +1366,15 @@ void GfxFrameout::kernelSetNowSeen(const reg_t screenItemObject) const {
 
 	ScreenItem *screenItem = plane->_screenItemList.findByObject(screenItemObject);
 	if (screenItem == nullptr) {
-		error("kSetNowSeen: Screen item %04x:%04x not found in plane %04x:%04x", PRINT_REG(screenItemObject), PRINT_REG(planeObject));
+		return false;
```

Comment:
Per IRC discussion, all SCI2/2.1early games + *some* early SCI2.1mid games (SQ6, maybe MGDX, probably not Shivers) will quit with an error if a ScreenItem is not found, but all other SCI2.1mid+ games do what this code does. We can error out in the kernel function if it seems to be necessary for those older games to run properly. So far it doesn’t seem to be necessary.